### DISCUSSION TITLE
Fortnite keyboard and mouse

### DISCUSSION THREAD

**Discussion by BigBlanchomme** [unknownskl/xbox-xcloud-client#1244] [Q&A]
The mouse sensitivity on fortnite is way too high and secondly playing with mouse and keyboard on games keeps the top menu with the net debug button still there, and not going, Pls help

**Comment by dark2377** [comment]
It's the same for Sea of Thieves, I tried lowering the in-game sensitivity but its still pretty high for me! And the visible top menu is very annoying.
